### PR TITLE
Add TC Tests

### PR DESCRIPTION
**Pull request by AdityaAtulTewari**

### CODE REVIEW

**Review by pingle14 — DISMISSED** [review]
Added tests for all TC implementations and Corrected test so would compile and run as intended

File: microbench/triangle-counting/CMakeLists.txt [hunk 1]
```diff
@@ -86,7 +86,7 @@ add_subdirectory(src)
 
 # tests
 option(MICROBENCH_TESTING "Testing for microbenchmarks" ON)
```
Comment (pingle14) [inline microbench/triangle-counting/CMakeLists.txt:88]: Enable testing for DRV-X

**Review by pingle14 — CHANGES_REQUESTED** [review]

File: microbench/triangle-counting/CMakeLists.txt (hunk 1 above)
Comment (AdityaAtulTewari) [reply microbench/triangle-counting/CMakeLists.txt:88]: what does this mean?

Comment (pingle14) [reply microbench/triangle-counting/CMakeLists.txt:88]: Currently I think TC tests are missing the test_dummy and test_driver stuff that got DRVX tests working on pando-lib-gal?

**Review by pingle14 — APPROVED** [review]

**Pull request merged**